### PR TITLE
UGENE-7080. Update CMake build files

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -47,28 +47,26 @@ if (CMAKE_COMPILER_IS_GNUCXX)
...
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GCC_WARNING_FLAGS}")
 endif ()
 
-# 3rd party libs
+# 3rd party libraries.
 add_subdirectory(src/libs_3rdparty/QSpec)
 add_subdirectory(src/libs_3rdparty/breakpad)
 add_subdirectory(src/libs_3rdparty/samtools)
 add_subdirectory(src/libs_3rdparty/sqlite3)
 add_subdirectory(src/libs_3rdparty/zlib)
```
Comment (evelinabis) [inline CMakeLists.txt:68]: Where is qscore?

Comment (yalgaer) [reply CMakeLists.txt:68]: Fixed. Thank you!

**Review by evelinabis — APPROVED** [review]

**Review by DmitriiSukhomlinov — CHANGES_REQUESTED** [review]
It's not possible to build UGENE with CMake on Windows now - CMakeList.txt files are configured incorrectly for this OS. Do you want to fix it in this issue of create another one?

**Comment by yalgaer** [comment]
> It's not possible to build UGENE with CMake on Windows now - CMakeList.txt files are configured incorrectly for this OS. Do you want to fix it in this issue of create another one?

You are right I have never tested Windows so far. I will return to Windows it later, once I will be fully satisfied with the Linux support

**Pull request merged**